### DISCUSSION TITLE
挂载的可选参数好像不多

### DISCUSSION THREAD

**Discussion by flyingnight** [yuudi/rclone-webui-angular#48] [Q&A]
我看网上rclone教程挂载到本地时，添加了很多参数，比如rclone mount Aliyun: /mnt/mydisk/Aliyun --config /root/.config/rclone/rclone.conf --header "Referer:" --multi-thread-streams 8 --buffer-size 512M --vfs-fast-fingerprint --vfs-cache-mode full --no-modtime --file-perms 0777 --copy-links --allow-other --allow-non-empty  --umask 000 --daemon
但是webui这边好像参数不多

**Comment by yuudi** [comment]
挂载自定义参数可以在0.8.0版本后设置，在创建挂载点后，展开高级设置，然后用json格式进行配置
